content/javascript/concepts/objects/terms/getOwnPropertyDescriptor/getOwnPropertyDescriptor.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Object.getOwnPropertyDescriptor()' 
+Description: 'This is an object that enables access to full information on a property.' 
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Web Development'
+  - 'Computer Science'  
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Objects'
+  - 'Methods'  
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `Object.getOwnPropertyDescriptor()` method is a built-in object that enables access to full information on a property and  returns a property descriptor of the given property if it exists on the object, otherwise `undefined`.
```

Comment:
```suggestion
The **`Object.getOwnPropertyDescriptor()`** method is a built-in object that enables access to full information about a property and returns a property descriptor of the given property if it exists on the object, otherwise it returns `undefined`.
```
Remember that it only focuses on own properties directly defined on the object itself. This means it doesn't access inherited properties from the object's prototype chain. Try to add this description of the method in the content (Don't copy this line directly, write it in your own words that fits in the current sentence).